### ISSUE TITLE
Miner rx/0 connecting to 5443 and 4443 ports

### ISSUE THREAD

**Issue by semen8989** [UnamSanctam/SilentCryptoMiner#2228]
I've found that miner is connecting to
[02.27 09:25:38] conhost.exe (System) - 193.176.87.178:4443
[02.27 09:25:38] conhost.exe (System) - 172.94.15.211:5443
these addresses. what are these? Is it donation for dev or what?

**Comment by UnamSanctam** [comment]
They shouldn't be, are you sure those conhost.exe are the miner?

**Comment by semen8989** [comment]
> They shouldn't be, are you sure those conhost.exe are the miner? I can't see them when sniffing the miner traffic.

Yes, miner stops sending shares and connect to these ip's. They connect for like 10 minutes every hour and closes the connection. I sniffed with wireshark, the vm which miner is running is fresh and doesn't have any installed apps besides the miner itself.

**Comment by UnamSanctam** [comment]
Hmm, I have't seen them in my Wireshark, are you sure you're using the miner from here: https://github.com/UnamSanctam/SilentCryptoMiner/releases?

**Comment by semen8989** [comment]
> Hmm, I have't seen them in my Wireshark, are you sure you're using the miner from here: https://github.com/UnamSanctam/SilentCryptoMiner/releases?

yes, I am 100% sure.
These two ip's detected as malware from websites haha
Try to run wireshark like a 2-3 hours.

**Comment by Chainski** [comment]
![image](https://github.com/UnamSanctam/SilentCryptoMiner/assets/96607632/4b9421fe-c6f1-4249-93af-53334e4727a9)

I hope the dev continue to steal from you guys.

**Comment by Chainski** [comment]
> I've found that miner is connecting to [02.27 09:25:38] conhost.exe (System) - 193.176.87.178:4443 [02.27 09:25:38] conhost.exe (System) - 172.94.15.211:5443 these addresses. what are these? Is it donation for dev or what?

config.sanctam.su:5443

**Comment by Chainski** [comment]
> > I've found that miner is connecting to [02.27 09:25:38] conhost.exe (System) - 193.176.87.178:4443 [02.27 09:25:38] conhost.exe (System) - 172.94.15.211:5443 these addresses. what are these? Is it donation for dev or what?
> 
``` 4Aw8Echp2Hrhc5ussZ5cX1bKS6AFJUqFMJH9373M819NCLMVs4DctwGgtTg1ixc8oqVhZNeKCSTS776xoihXmX8SNYx7vtv.RIG-3028```
